.changeset/curvy-cheetahs-collect.md

```diff
@@ -0,0 +1,6 @@
+---
+"@frontify/guideline-blocks-settings": patch
```

Comment:
```suggestion
"@frontify/guideline-blocks-settings": minor
```

Reply:
is there a guideline to what it takes for a change to be minor instead of patch?

Reply:
Basically this one: https://semver.org/